app/controllers/idv/socure/document_capture_controller.rb

```diff
@@ -117,6 +117,7 @@ def socure_errors_presenter(result)
           error_code: error_code_for(result),
           remaining_attempts:,
           sp_name: decorated_sp_session&.sp_name || APP_NAME,
+          issuer: decorated_sp_session&.sp_issuer,
```

Comment:
we do not appear to be calling this method `#socure_errors_presenter` ... rid of it?

Reply:
Done in 5b7562ed57